### DISCUSSION TITLE
Cannot hide unknown username output

### DISCUSSION THREAD

**Discussion by eoksum** [zitadel/zitadel#8708] [Q&A] [answered]
Hi everyone!

Im currently using Zitadel with Netbird and would like to hide the "object not found" error to prevent username guessing. I've enabled ignore unknown usernames checkbox but when I try to login with a unknown username I still get the "object not found" error on login screen.

**Comment by hifabienne** [comment]
Can you please show the screenshot of the error? Is there some more information?
And did you enable the setting on the organization where the user belongs, respecitvely also on the default orgnization?

**Reply by eoksum** [reply]
Dear @hifabienne 
I did enable the setting on the correct organization. I actually only have a single organization.
I'm sharing the screenshots here:

The error I'm receiving when I try to login with random username:
<img width="179" alt="zitadel2" src="https://github.com/user-attachments/assets/f20ebf9f-58fb-4f70-90db-030f9f24e58a">
The setting I enabled:
<img width="154" alt="zitadelconf" src="https://github.com/user-attachments/assets/87c07169-4501-4e54-943d-999332b8c5eb">

**Reply by hifabienne** [reply]
Do you run on a selfhosting instance? If so on which version?
This looks like an error we have fixed some time ago

**Reply by livio-a** [reply]
Apparently Netbird's script uses an outdated version (currently: 2.54.3: https://github.com/netbirdio/netbird/blob/8934453b30309e508df09236ac102c0537259291/infrastructure_files/getting-started-with-zitadel.sh#L876). We've backported multiple fixes including security fixes to this version. Please update to at least 2.54.10

**Reply by eoksum** [reply]
Dear @livio-a 
Can I update Zitadel without reinstalling Netbird? and if so, how can I do this please?

**Reply by livio-a** [reply]
I quickly scanned their script. From what i see, you should get a docker-compose file, where you should be able to update the version of the zitadel image.

**Reply by eoksum** [reply]
Dear @livio-a 

Sorry for making you babysit me here but I'm not a devops guy and don't really have much knowledge regarding Docker. I've found docker compose file in /root/docker-compose.yml and edited Zitadel container version to 2.54.10. Now how can I make docker update that container to this version?

**Reply by livio-a** [reply, marked as answer]
Using the following commands you'll be able to replace the zitadel image:
`docker compose pull` -> you should see the new image getting pulled
`docker compose up -d` will start all services / restart zitadel with the new image

you probably need to reference the compose file in your case using the `-f <path>` flag

**Reply by eoksum** [reply]
> Using the following commands you'll be able to replace the zitadel image: `docker compose pull` -> you should see the new image getting pulled `docker compose up -d` will start all services / restart zitadel with the new image
> 
> you probably need to reference the compose file in your case using the `-f <path>` flag

That was the trick! Thank you for your support.
For future reference,
1- I edited /root/docker-compose.yml file and changed Zitadel's version to the provided version
2- I executed commands shared by @livio-a 

Now everything seem to work as expected.
